README.md

```diff
@@ -1,11 +1,10 @@
 Identity-IdP (Upaya)
 ====================
 
-[![Build Status](https://travis-ci.org/18F/identity-idp.svg?branch=master)](https://travis-ci.org/18F/identity-idp)
-[![Code Climate](https://codeclimate.com/github/18F/identity-idp/badges/gpa.svg)](https://codeclimate.com/github/18F/identity-idp)
-[![Test Coverage](https://codeclimate.com/github/18F/identity-idp/badges/coverage.svg)](https://codeclimate.com/github/18F/identity-idp/coverage)
+[![Build Status](https://circleci.com/gh/18F/identity-idp.svg?style=svg)](https://circleci.com/gh/18F/identity-idp)
+[![Code Climate](https://api.codeclimate.com/v1/badges/e78d453f7cbcac64a664/maintainability)](https://codeclimate.com/github/18F/identity-idp/maintainability)
```

Comment:
mmh for some reason the `code climate` and `test coverage` badges aren't rendering in my browser

Reply:
Try again? I saw the same thing but now it seems to be working. It looks like GitHub automatically fetches external images and stores them on its own servers, but for some reason the new image URL was returning a 502. But now it's rendering for me.

Reply:
oh yea it renders for me now too!